internal/proxy/task_index.go

```diff
@@ -160,6 +160,8 @@ func (cit *createIndexTask) parseIndexParams() error {
 			if exist && !validateArithmeticIndexType(specifyIndexType) {
 				return merr.WrapErrParameterInvalid(DefaultArithmeticIndexType, specifyIndexType, "index type not match")
 			}
+		} else if typeutil.IsBoolType(cit.fieldSchema.DataType) {
+			// TODO: should we check something?
```

Comment:
how about :
if !exist {
				indexParamsMap[common.IndexTypeKey] = INVERTED_INDEX_TYPE
			}
			if exist && specifyIndexType!= INVERTED_INDEX_TYPE) {
				return merr.WrapErrParameterInvalid(INVERTED_INDEX_TYPE, specifyIndexType, "index type not match")
			}